### ISSUE TITLE
BertForTokenClassification for NER . what is the conclusion of this output ?

### ISSUE THREAD

**Issue by ajbot2019** [google-research/bert#926]
Hi ,
Im trying to perform NER using BertForTokenClassification .I saw this sample code in transformers GIT page.

from transformers import BertForTokenClassification
tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
model = BertForTokenClassification.from_pretrained('bert-base-uncased')
input_ids = torch.tensor(tokenizer.encode("Hello, my dog is cute")).unsqueeze(0) # Batch size 1
labels = torch.tensor([1] * input_ids.size(1)).unsqueeze(0) # Batch size 1
print(labels)
outputs = model(input_ids, labels=labels)
loss, scores = outputs[:2]

output loss:
tensor(0.5975, grad_fn=)

output scores:

tensor([[[-0.1622, 0.1824],
[-0.1552, -0.0534],
[-0.3032, -0.1166],
[-0.2453, -0.1182],
[-0.4388, -0.1898],
[-0.3159, -0.1067]]], grad_fn=)

1.When i printed the loss and score i got below values .Now how should i infer this output ? what dose these value represent for performing NER ? what should i do to get the NER tags for the sentence "Hello, my dog is cute" .

2.i referred few NER codes in GIT using BERT and they have humongous line of code written for performing the NER . Is there any simple way to perform NER using bert ? like how Flair library has very simple method for performing the NER task ?
